src/Components/LogTable.css.d.ts

```diff
@@ -1,30 +1,30 @@
 declare namespace LogTableCssNamespace {
-	export interface ILogTableCss {
-		active: string;
-		app: string;
-		column: string;
-		copy: string;
-		copyMenu: string;
-		empty: string;
-		highlight: string;
-		level: string;
-		level_1: string;
-		level_2: string;
-		level_3: string;
-		level_4: string;
-		"log-settings-toggle": string;
-		logs: string;
-		message: string;
-		relative: string;
-		row: string;
-		smallHeader: string;
-		time: string;
-	}
+  export interface ILogTableCss {
+    active: string;
+    app: string;
+    column: string;
+    copy: string;
+    copyMenu: string;
+    empty: string;
+    highlight: string;
+    level: string;
+    level_1: string;
+    level_2: string;
```

Comment:
Please check that also: property 'focus-within' or 'focus-visible' was removed, but file still has changes due to indent (tabs or spaces?)